packages/cli/src/utils/command.ts

```diff
@@ -111,6 +115,11 @@ export const Flags = {
     description: 'Phone Number in E164 Format',
     helpValue: '+14152223333',
   }),
+  proofOfPossession: flags.build({
```

Comment:
Love this! Can we use this flag type in other commands where relevant?

Reply:
Absolutely, its specifically for a 65 byte signature as you can see it even validates that it is 65-bytes (`POP_SIZE`)

```
const parseProofOfPossession: ParseFn<string> = (input) => {
  return parseBytes(input, POP_SIZE, `${input} is not a proof-of-possession`)
}
```